### PR TITLE
update Annotations description about audit.Event

### PR DESCRIPTION
**Pull request by CaoShuFeng**
ref: https://github.com/kubernetes/kubernetes/pull/58679#discussion_r209375632

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/assign @liggitt 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/apis/audit/types.go [hunk 1]
```diff
@@ -132,9 +132,10 @@ type Event struct {
 
 	// Annotations is an unstructured key value map stored with an audit event that may be set by
 	// plugins invoked in the request serving chain, including authentication, authorization and
-	// admission plugins. Keys should uniquely identify the informing component to avoid name
-	// collisions (e.g. podsecuritypolicy.admission.k8s.io/policy). Values should be short. Annotations
-	// are included in the Metadata level.
+	// admission plugins. But it does not contain the annotations of the submitted object.
```
Comment (liggitt) [inline staging/src/k8s.io/apiserver/pkg/apis/audit/types.go]: suggest "Note that these annotations are for the audit event, and do not contain the metadata.annotations of the submitted object."

Comment (liggitt) [reply staging/src/k8s.io/apiserver/pkg/apis/audit/types.go]: also update v1alpha1 and v1beta1 audit types?

Comment (CaoShuFeng) [reply staging/src/k8s.io/apiserver/pkg/apis/audit/types.go]: Done.

**Pull request merged**